README.md

```diff
@@ -4,11 +4,15 @@
 It is recommended to have first run the [Kids First Germline Annotation Workflow](https://github.com/kids-first/kf-germline-workflow/blob/v0.4.4/docs/GERMLINE_SNV_ANNOT_README.md) first.
 
 ## Pathogenicity Preprocessing Workflow
-This workflow uses the prerequisite input to run the InverVar workflow and autoPVS1 tool.
-Recommended inputs:
- - `annovar_db`: Annovar Database with at minimum required resources to InterVar. Need to use [annovar download commands](https://annovar.openbioinformatics.org/en/latest/user-guide/download/) to get the following:
+This workflow uses the prerequisite input to run the InterVar workflow and autoPVS1 tool.
+The major pieces of software being used are:
+ - ANNOVAR latest: The software has no versioning, but references do. See `annovar_db` section in [Recommended inputs](#recommended-inputs)
+ - InterVar v2.2.1
+ - AutoPVS1 v2.0a: Modified from the original to fit annotated KF vcf output. See [README for autoPVS1](autopvs1/README.md) for details
```

Comment:
so now will be 1.0.0 (but modified from AutoPVS1 v2.0)?

Reply:
Correct, I will update the readme after I peel it off. It'll be cleaner and better for maintenance if needed. I'll ping again after for hopefully a final review